src/bin/vip-dev-env-logs.js

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env node
+
+/**
+ * @flow
+ * @format
+ */
```

Comment:
```suggestion
// @flow
// @format
```

JSDoc does not like these tags.